src/keywords.YAML-tmLanguage

```diff
@@ -44,7 +44,7 @@ repository:
                 match: \b(constructor|init)\b
                 name: entity.name.function.constructor
             -
-                match: \b(companion|object)\b
+                match: \b(companion|object|data\s+object)\b
```

Comment:
Thanks for your contribution. I'm assuming that this change is motivated by https://kotlinlang.org/docs/object-declarations.html#data-objects. I'd rather have `data` tagged independently as a storage modifier than make `data object` as a single keyword. I think the correct place to make the adjustment would be making the change [here](https://github.com/nishtahir/language-kotlin/blob/0553adec020f350bb726063a4bf85f587aac673d/src/classes.YAML-tmLanguage#L5)

Reply:
Ok, I'll rework :)

Reply:
So I did so, hopefully, it is correct now.

Reply:
@nishtahir please :)